src/chrome/content/rules/umeng.xml

```diff
@@ -0,0 +1,40 @@
+<!--
+	Mismatch:
+	- opentest.rest.api.wsq.umeng.com
+	- tip.umeng.com
+
+	Redirects to http
+	- bbs.umeng.com
+	- blog.umeng.com
+	- bugbeta.umeng.com
+	- im.umeng.com
+	- message.umeng.com
+	- push.umeng.com
+	- track.umeng.com
+	- ubdc2016.umeng.com
+
+	Refused:
+	- huodong.umeng.com
+-->
+
+<ruleset name="umeng.com">
+	<target host="umeng.com" />
+	<target host="beta.umeng.com" />
+	<target host="dev.umeng.com" />
+	<target host="dplus.umeng.com" />
+	<target host="i.umeng.com" />
+	<target host="index.umeng.com" />
+	<target host="mobile.umeng.com" />
+	<target host="s.umeng.com" />
+	<target host="salon.umeng.com" />
+	<target host="static.umeng.com" />
+	<target host="wsq.umeng.com" />
+	<target host="api.wsq.umeng.com" />
+	<target host="www.umeng.com" />
+	
+	<!--mixed content:-->
+	<exclusion pattern="^http://salon.umeng.com/$" />
```

Comment:
There are various deep links with this problem, you should find a better regex and test urls for it or exclude it completely.